### ISSUE TITLE
Sounding Pokemon cries

### ISSUE THREAD

**Issue by shigeyukey** [Unlucky-Life/ankimon#11]
You may have already considered it, I am thinking it would be interesting to play Pokemon cries.
(e.g. when a Pokemon emerges, or when a user selects a collection of Pokemon).

The audio files are available from pokemonshowdown as well.(But the functionality can be complicated, so it may not be important.)
https://play.pokemonshowdown.com/audio/cries/

And I think it would be nice to have an explanation to enable Gifs for Pokemon. (I just noticed that there are animations! Great!👍)

**Comment by Unlucky-Life** [comment]
yes :) the gifs explanation i will integrate ! But the whole addon was alot of work and still in development and a few bugs aswell :). But in general yes move sounds and pokemon cries i would be into adding in !
